index.html

```diff
@@ -0,0 +1,154 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Capstone project</title>
+    <link rel="stylesheet" href="style.css">
+    <script src="script.js" defer></script>
+</head>
+<body>
+    <section class="header-section">
+        <div class="main-containers">
+            <nav class="inner-header">
+                <li class="inner-list"><img src="images/facebook.png" alt=""></li>
+                <li class="inner-list"><img src="images/twitter.png" alt=""></li>
+                <li class="inner-list"><a href="#">English</a></li>
+                <li class="inner-list"><a href="#">My page</a></li>
+                <li class="inner-list"><a href="#">Logout</a></li>
+            </nav>
```

Comment:
- [ ] Kindly follow the given template by removing the dark navigation for the mobile version.

Your design
![image](https://user-images.githubusercontent.com/81932478/196736825-ef46517f-6635-4908-a2db-4a0b4e3c8a38.png)

Given design
![image](https://user-images.githubusercontent.com/81932478/196737003-b09a0010-6f90-4e1f-b21b-d8f30cabd26a.png)